src/components/Lists/Lists.css

```diff
@@ -42,31 +42,42 @@
   text-align: center;
   font-size: 20px;
   color: black;
-  box-shadow: 3px 2px black;
 }
 
 .shoppingList {
   width: 100%;
 }
 
 .listTitle {
-  font-family: var(--titleFonts);
-  font-size: 1.5em;
-  margin-bottom: 1vh;
+  font-family: var(--secondaryFonts);
+  font-size: 18px;
+  margin-bottom: 0.5vh;
+  padding-top: 1vh;
   color: black;
   font-weight: bold;
+  text-transform: capitalize;
 }
 
 .storeDetails {
   font-family: var(--secondaryFonts);
-  margin-bottom: 1vh;
+  font-size: 1rem;
+  margin-bottom: 0.25vh;
 }
 
 .distanceFromStore {
   font-family: var(--secondaryFonts);
-  margin-bottom: 1vh;
+  font-size: 1rem;
+  margin-bottom: 0.5vh;
+}
+
+.fa-trash-alt {
```

Comment:
*please* stop assigning styles to the font awesome icons directly. Add a new class for your particular usage. Otherwise, if someone else uses this icon then this style choice is added too!